bluebrain/documentation/setup_personal.md

```diff
@@ -25,56 +34,70 @@ conjunction with Apple's CLang:
 
     $ brew install gcc
 
+### Building software on Ubuntu
+
+Ubuntu / Debian have a habit of being somewhat _special_, patching upstream
+projects in unexpected ways.
+Consider yourself warned to not rely on system packages and defer to
+Spack-installed ones when needed.
+
+First, ensure that the essential packages to building stuff are installed:
+
+    $ sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+    $ sudo apt update
+    $ sudo apt install build-essential gcc-11 g++-11 gfortran-11
+
+If Python 3 is not your default, tell Ubuntu to use a newer one, e.g.,
+Python 3.8 by setting the default `python`:
+
+    $ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1
+
+To check that we are using Python 3 as `python`:
+
+    $ sudo update-alternatives --config python
+    There is only one alternative in link group python
+    (providing /usr/bin/python): /usr/bin/python3.8. Nothing to configure.
+
+## Spack Setup
+
 Now clone our version of Spack and find compilers and external packages:
 
     $ git clone -c feature.manyFiles=true https://github.com/BlueBrain/spack.git
     $ . spack/share/spack/setup-env.sh
     $ spack compiler find
     $ spack external find
 
```

Comment:
Still checking stuff but I think that the steps setup should go into steps and not in the general spack

Reply:
Sure